From 6205d99c717ce06a51ecf7921ff7e975229a8725 Mon Sep 17 00:00:00 2001
From: Vidya Velumuri <vidya.velumuri@cavium.com>
Date: Fri, 19 Oct 2018 10:04:21 +0300
Subject: [PATCH 0327/1051] octeontx-dpi: Use of actual correct macro
 DPI_DMA_REQQ_INT

Replaced the use of DPI_REQQ_INT with DPI_DMA_REQQ_INT
at proper places.

Signed-off-by: Vidya Velumuri <vidya.velumuri@cavium.com>
Signed-off-by: Yury Norov <ynorov@marvell.com>
Signed-off-by: Yury Norov <ynorov@caviumnetworks.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../net/ethernet/cavium/octeontx-83xx/dpipf_main.c   | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c b/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c
index 4e214a648589..bd7e02069f03 100644
--- a/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c
+++ b/drivers/net/ethernet/cavium/octeontx-83xx/dpipf_main.c
@@ -418,14 +418,17 @@ static irqreturn_t dpi_pf_intr_handler (int irq, void *dpi_irq)
 			i, reg_val);
 		dpi_reg_write(dpi, DPI_DMA_CCX_INT(i), 0x1ULL);
 	} else if (i < DPI_DMA_INT_REG) {
-		reg_val = dpi_reg_read(dpi, DPI_REQQX_INT(i - DPI_REQQ_INT));
-		dev_err(&dpi->pdev->dev, "DPI_REQQ_INT raised for q:%d: 0x%016llx\n",
+		reg_val = dpi_reg_read(
+			dpi, DPI_REQQX_INT(i - DPI_DMA_REQQ_INT));
+		dev_err(&dpi->pdev->dev,
+			"DPI_REQQ_INT raised for q:%d: 0x%016llx\n",
 			(i - 0x40), reg_val);
 
-		dpi_reg_write(dpi, DPI_REQQX_INT(i - DPI_REQQ_INT), reg_val);
+		dpi_reg_write(
+			dpi, DPI_REQQX_INT(i - DPI_DMA_REQQ_INT), reg_val);
 
 		if (reg_val & (0x71ULL))
-			dpi_queue_reset(dpi, (i - DPI_REQQ_INT));
+			dpi_queue_reset(dpi, (i - DPI_DMA_REQQ_INT));
 	} else if (i == DPI_DMA_INT_REG) {
 		reg_val = dpi_reg_read(dpi, DPI_INT_REG);
 		dev_err(&dpi->pdev->dev, "DPI_INT_REG raised: 0x%016llx\n",
@@ -839,4 +842,3 @@ static void __exit dpi_cleanup_module(void)
 
 module_init(dpi_init_module);
 module_exit(dpi_cleanup_module);
-
-- 
2.17.1

